api/test/wfl/unit/workloads_test.clj

```diff
@@ -5,13 +5,13 @@
   (:import (clojure.lang ExceptionInfo)))
 
 (deftest test-saved-before?
-  "in version 0.4.0, wgs workloads were serialized using CromwellWorkload table"
-  (is (not (workloads/saved-before? "0.4.0" {:version "0.4.0"})))
-  (is (not (workloads/saved-before? "0.4.0" {:version "1.0.0"})))
-  (is (not (workloads/saved-before? "0.4.0" {:version "0.4.1"})))
-  (is (workloads/saved-before? "0.4.0" {:version "0.3.8"}))
-  (is (workloads/saved-before? "0.4.0" {:version "0.0.0"}))
-  (is (thrown? ExceptionInfo (workloads/saved-before? "0.4" {:version "0.0.0"}))))
+  (testing "in version 0.4.0, wgs workloads were serialized using CromwellWorkload table"
```

Comment:
I really don't like the "testing blah blah blah" in tests like this. It adds nothing. Rename the test if you think it's not clear.
Also, this change was not limited to WGS

Reply:
I just wrapped what looked like an attempt at a docstring.
My goal was to reduce reports from the linter without changing behavior.
Except where a linter discovered a problem: then I tried to fix the problem.